pkg/keyspace/tso_keyspace_group.go

```diff
@@ -637,6 +633,9 @@ func buildSplitKeyspaces(
 		newKeyspaceMap = make(map[uint32]struct{}, newNum)
 	)
 	for _, keyspace := range old {
+		if keyspace == utils.DefaultKeyspaceID {
```

Comment:
We can't add this check, because it means that we can't split default keyspace group now.